src/test/homework1/CustomListTest.java

```diff
@@ -0,0 +1,87 @@
+package homework1;
+
+
+import static org.junit.Assert.assertNotEquals;
+
+public class CustomListTest {
+
+    private CustomList customList;
+
+    @org.junit.Before
+    public void init() {
+        customList = new CustomList();
+    }
+
+
+    @org.junit.Test
+    public void add() {
+        customList.addFirst(1);
+        customList.add(1, 2);
+        customList.addLast(4);
+        assertEquals(customList.getIndex(0).value, 1);
+        assertEquals(customList.getIndex(1).value, 2);
+        assertEquals(customList.getIndex(2).value, 4);
+
+    }
+
+    private void assertEquals(int value, int i) {
```

Comment:
?

Reply:
It was removed, thanks for the observations